tests/unit/compat/mock.py

```diff
@@ -1,109 +1,30 @@
-# Copyright (c) 2014, Toshio Kuratomi <tkuratomi@ansible.com>
+# Copyright (c) Ansible Project
 # GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-# Make coding more python3-ish
+# Copied from https://github.com/ansible/ansible/blob/b86a18bd273499f5d10e581821a47571690660e1/test/units/compat/mock.py
+
+"""
+Compatibility shim for mock imports in modules and module_utils.
+This can be removed once support for Python 2.7 is dropped.
+"""
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
-'''
-Compat module for Python3.x's unittest.mock module
-'''
-import sys
-
-# Python 2.7
-
-# Note: Could use the pypi mock library on python3.x as well as python2.x.  It
-# is the same as the python3 stdlib mock library
-
 try:
-    # Allow wildcard import because we really do want to import all of mock's
-    # symbols into this compat shim
-    # pylint: disable=wildcard-import,unused-wildcard-import
-    from unittest.mock import *
+    from unittest.mock import (
```

Comment:
Why not keep the wildcard import here (and below)?

Reply:
This is what mattclay chose to do upstream. I could revert this part if you'd like.

Reply:
I would keep it. In fact I would probably keep the old file, except the bottom part for old Python 3.x versions. (I.e. basically what community.internal_test_tools contains.)